### PR TITLE
Fix up NERSC Squid resources (provided by Cory Snavely) (FD #72189)

### PR DESCRIPTION
**Pull request by matyasselmeci**

### CODE REVIEW

**Comment by matyasselmeci** [comment]
One sec, let me check something with the admin

File: topology/National Energy Research Scientific Computing Center/NERSC/NERSC-Infra.yaml [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
+GroupDescription: NERSC Infrastructure (off-HPC-platform)
 
 GroupID: 1348
 
 Resources:
-  NERSC-Non-HPC-Squid1:
+  NERSC-Infra-Squid1:
     Active: false
-    Description: This is a Frontier-Squid instance primarily for non-HPC systems at NERSC that need access to CVMFS
+    Description: This is a Frontier-Squid instance situated off the HPC platform. It is primary for all Frontier Cache use, primary for off-HPC-platform CVMFS clients, and secondary (failover) for on-HPC-platform CVMFS clients.
```
Comment (brianhlin) [inline topology/National Energy Research Scientific Computing Center/NERSC/NERSC-Infra.yaml:11]:
```suggestion
    Description: >-
      This is a Frontier-Squid instance situated off the HPC platform. It is primary for all Frontier Cache
      use, primary for off-HPC-platform CVMFS clients, and secondary (failover) for on-HPC-platform CVMFS clients.
```

File: topology/National Energy Research Scientific Computing Center/NERSC/NERSC-Infra.yaml [hunk 2]
```diff
@@ -37,9 +37,9 @@ Resources:
       NERSC: 100
 
 
-  NERSC-Non-HPC-Squid2:
+  NERSC-Infra-Squid2:
     Active: false
-    Description: This is a Frontier-Squid instance primarily for non-HPC systems at NERSC that need access to CVMFS
+    Description: This is a Frontier-Squid instance situated off the HPC platform. It is primary for all Frontier Cache use, primary for off-HPC-platform CVMFS clients, and secondary (failover) for on-HPC-platform CVMFS clients.
```
Comment (brianhlin) [inline topology/National Energy Research Scientific Computing Center/NERSC/NERSC-Infra.yaml:42]:
```suggestion
    Description: >-
      This is a Frontier-Squid instance situated off the HPC platform. It is primary for all Frontier Cache
      use, primary for off-HPC-platform CVMFS clients, and secondary (failover) for on-HPC-platform CVMFS clients.
```

**Review by brianhlin — APPROVED** [review]
I wasn't kidding about style!

**Pull request merged**